python/cudf/cudf/tests/test_serialize.py

```diff
@@ -327,6 +349,27 @@ def test_serialize_decimal_columns(data):
     assert_eq(recreated, df)
 
 
+@pytest.mark.parametrize(
+    "fields",
+    [
+        {"a": np.dtype(np.int64)},
+        {"a": np.dtype(np.int64), "b": None},
+        {
+            "a": cudf.ListDtype(np.dtype(np.int64)),
+            "b": cudf.Decimal64Dtype(1, 0),
+        },
+        {
+            "a": cudf.ListDtype(cudf.StructDtype({"b": np.dtype(np.int64)})),
+            "b": cudf.ListDtype(cudf.ListDtype(np.dtype(np.int64))),
+        },
+    ],
+)
+def test_serialize_struct_dtype(fields):
```

Comment:
I think we decided to move towards putting tests that exclusively test functionality for a specific extension type to the `.py` files that have that name, to make things a little easier to find/organize. So this might be relocated to `test_struct.py`.